SQLQuery_1.sql

```diff
@@ -0,0 +1,49 @@
+
+CREATE DATABASE FindYourRecipe;
+CREATE TABLE Recipes 
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Title VARCHAR(50),
+    Instructions VARCHAR(MAX),
+    Category VARCHAR(30),
+    Dificulty VARCHAR(30)
```

Comment:
`Category` and `Difficulty` should be stored in two different tables which should have a relationship to this one. Remember the C# enum structure that only stores a specific set of values? This is what we want here because I assume there will be an exact set of difficulties (example: simple, medium, complex) and an exact set of categories to choose from.